src/components/link/link.component.tsx

```diff
@@ -84,14 +85,19 @@ export const Link = React.forwardRef<
     const [hasFocus, setHasFocus] = useState(false);
     const l = useLocale();
     const { inMenu } = useContext(MenuContext);
+    const isAButton = onClick && !href;
```

Comment:
comment: I think it might be overkill to add these additional checks here. We could probably just add the same as in the other components and not really care if it's a button etc